tasks/extensions/postgis.yml

```diff
@@ -12,3 +12,25 @@
     - "postgresql-{{postgresql_version}}-postgis-scripts"
   notify:
     - restart postgresql
+  when: ansible_os_family == "Debian"
+
+- name: PostgreSQL | Extensions | Make sure the postgis extensions are installed | RedHat
+  yum:
+    name: "{{item}}"
+    state: present
+  with_items:
+    - "postgis2_{{postgresql_version_terse}}"
+    - "postgis2_{{postgresql_version_terse}}-client"
+    - "postgis2_{{postgresql_version_terse}}-utils"
+  notify:
+    - restart postgresql
+  when: ansible_os_family == "RedHat"
+
+- name: PostgreSQL | Extensions | Make sure the postgis development files are installed | RedHat
+  yum:
+    name: "{{item}}"
+    state: present
+  with_items:
+    - "postgis2_{{postgresql_version_terse}}-devel"
+    - "postgis2_{{postgresql_version_terse}}-docs"
+  when: (ansible_os_family == "RedHat") and (postgresql_ext_install_dev_headers)
```

Comment:
when: ansible_os_family == "RedHat" and {{ postgresql_ext_install_dev_headers | default(false) }}

Reply:
I think the dev_headers was fixed by #327 , but the PostGIS dev headers aren't done.  It may be best to add it to #310?

Reply:
sure you can do that